.github/workflows/megatron.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  workflow_dispatch:
+    inputs:
+      base-image:
+        description: 'Base image to use'
+        required: true
+  push:
+    paths:
+      - "megatron/**"
+      - ".github/workflows/megatron.yml"
+      - ".github/workflows/build.yml"
+
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yml
+    secrets: inherit
+    with:
+      image-name: megatron
+      folder: megatron
+      build-args: |
+        BASE_IMAGE=${{ inputs.base-image || 'ghcr.io/coreweave/ml-containers/torch-extras:bfe03aa-nccl-cuda12.4.1-ubuntu22.04-nccl2.21.5-1-torch2.4.0-vision0.19.0-audio2.4.0'}}
```

Comment:
Do we want to parameterize this?

Reply:
It seems to be impossible to do this according to https://github.com/orgs/community/discussions/26322